alerts_backend/python/app.py

```diff
@@ -153,10 +174,10 @@ def get_alerts_not_from_app(existing_alert_ids: Set[int]):
     logger.info(f"Making AeroAPI request to GET {api_resource}")
     result = AEROAPI.get(f"{AEROAPI_BASE_URL}{api_resource}")
     if not result:
-        return None
+        return []
```

Comment:
minor change in function doc string to indicate empty list return instead of None